lib/buffer.js

```diff
@@ -65,8 +65,15 @@ const zeroFill = bindingObj.zeroFill || [0];
 
 function createBuffer(size, noZeroFill) {
   if (noZeroFill)
-    zeroFill[0] = 0;  // Reset by the runtime.
-  const ui8 = new Uint8Array(size);
+    zeroFill[0] = 0;
+
+  try {
+    var ui8 = new Uint8Array(size);
+  } finally {
+    if (noZeroFill)
```

Comment:
'Needed', no, but I added it for symmetry with the check above and because it saves a bounds check.